network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -239,16 +247,25 @@ def entries_by_topic(self, request, topic, *args, **kwargs):
         return IndexPage.serve(self, request, *args, **kwargs)
 
     @route(r'^search/')
-    def search(self, request):
+    def search(self, request: 'HttpRequest') -> 'HttpResponse':
         """Render search results view."""
+
+        query = request.GET.get('q', '')
+
+        context_overrides = {
+            'index_title': 'Search',
+            'entries': self.get_search_entries(query=query)[:6],
+            'query': query,
+        }
+
```

Comment:
**Suggestion**

Send an extra `no_results = f'no results found' for {query}'` variable through and check `if entries` in the template. At the moment there is no messaging for the user if there are no results

Reply:
Good catch. There is #8776 to add a template for this, but I could definitely handle the case already and just fill in the template 👍